metabox/metabox/metabox-provider/units/basic-tps.pxu

```diff
@@ -45,3 +45,9 @@ include:
  config-environ-source
  config-environ-vars
  config-environ-case
+
+unit: test plan
+id: installation_verify
```

Comment:
I don't like how the thing evolves on each layer.
All starts with "dependency-import" then it becomes a "dependency-test.py", then it becomes "dependency_installation" job and is included in "Verify Checkbox installation" test plan that has an id "installation-verify".  Everything that's done is the original "dependency-import" test. Introducing those intermediate names is an unnecessary burden for the reader I think.

Reply:
Ok the dependency dance (import/verify etc.) is not ok, changing that.

The test plan is there so that if we come up with some new way of verifying the installation we can just add to that test plan and the scenario will automatically run any new verification. We may want to verify the environment of snap for example? Something like that.

Reply:
but the modification to use the test plan is to edit the scenario and create the test plan, while with this we have 2 lines more and we get the same result, it is a minuscule amount of code for a pretty nice future result